azuredevops.go

```diff
@@ -75,11 +76,11 @@ func (m importModule) runAzureDevOpsHandler(c *gin.Context) {
 	}
 
 	importer := importer.NewAzureImporter(c, &client)
-	token := wharfapi.Token{
+	token := response.Token{
 		TokenID:  i.TokenID,
 		Token:    i.Token,
 		UserName: i.UserName}
-	provider := wharfapi.Provider{
+	provider := response.Provider{
 		ProviderID: i.ProviderID,
 		Name:       providerName,
 		URL:        i.URL,
```

Comment:
Changed to `response` version instead of `request` version because `request` does not have an ID field, and splitting import/refresh behavior is out-of-scope for this PR IMO.

Reply:
Suggest instead to make use of a new type then instead, something like:

```go
type tokenReqWithID struct {
    request.Token
    ID uint `json:"id"`
}

type providerReqWithID struct {
    request.Provider
    ID uint `json:"id"`
}
```

Not my proudest name suggestion, but I think the concept is better than abusing a response type for a request payload